ocs_ci/ocs/constants.py

```diff
@@ -1205,6 +1205,14 @@
 
 # Config related constants
 config_keys_patterns_to_censor = ["passw", "token", "secret", "key", "credential"]
+config_keys_expressions_to_censor = [
+    "AccountKey",
+    "AccountName",
+    "ApplicationID",
+    "ApplicationSecret",
+    "LogsAnalyticsWorkspaceID",
+    "TenantID",
+]
```

Comment:
Cant we add these values as part of `config_keys_patterns_to_censor`

Reply:
That's the way I implemented it at first and it does technically work, but including full expressions along side pattern keywords like "secret" and "key" felt semantically wrong. This differentiation also prevents any potential side-effect censoring (if for example we include "AccountKey" under the patterns list then we would also censor a theoretical non-confidential key like "AccountKeyEncryptionType").

Reply:
Thanks for the clarification